pylint/config.py

```diff
@@ -417,10 +418,10 @@ def add_option_definitions(self, option_definitions):
             group, args, kwargs = self._convert_definition(option, definition)
             option_groups[group].append((args, kwargs))
 
-        for args, kwargs in option_groups['DEFAULT']:
+        for args, kwargs in option_groups['MASTER']:
```

Comment:
Old INI files had MASTER as the default section. That needs accounting for a few places in the code.